src/components/common/Message/index.js

```diff
@@ -0,0 +1,26 @@
+import './styles.scss';
+import classNames from 'classnames';
+import { useSelector } from 'react-redux';
+import { selectAuth } from 'services/auth/auth';
+
+const Message = ({ message: { content, date, user } }) => {
+  const { user: currentUser } = useSelector(selectAuth);
+
+  const ifSelf = (then, otherwise) => (user.id === currentUser.id ? then : otherwise);
+
+  const formatDate = date => {
+    const theDate = new Date(date);
+    return `${theDate.getHours() % 12}.${String(theDate.getMinutes()).padStart(2, '0')} ${
```

Comment:
WDYT if you remove this functiom and put it inside a `/utils` file and then import it here? Just to make it more readable

Reply:
Sounds good!